core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -586,3 +624,19 @@ <h3 ng-if="action === 'publicize_exploration'">Boost exploration ranking (as mod
     <button class="btn btn-danger" ng-click="reallyDelete()">Delete Exploration</button>
   </div>
 </script>
+
+<script>
+  function load() {
+    var explorationTitle = document.getElementById('explorationTitle').value;
+    var explorationObjective = document.getElementById("explorationObjective").value;
+    if( explorationTitle != ''){
+      var modalExplorationTitle = document.getElementById("modal-exploration-title");
+      modalExplorationTitle.innerHTML = explorationTitle;
```

Comment:
As a general rule, never set innerHTML. This can cause security issues due to XSS.